Recommended Ubuntu packages don't exist

https://github.com/scylladb/charybdefs/tree/c55ab834569d3979ec69c090b43df7c02286a61a#building

```
% sudo apt-get install g++ cmake fuse libfuse-dev libthrift-dev python-thrift
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package libthrift-dev
```

Ubuntu 16.04 does not have a package `libthrift-dev` as written in the README.

Hello,
Any update on the "libthrift-dev" package on ubuntu?
I get following error on executing cmake

`
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.26")
-- Performing Test FUSE_CFLAGS_CHECK
-- Performing Test FUSE_CFLAGS_CHECK - Success
-- Found FUSE: /usr/lib/x86_64-linux-gnu/libfuse.so (found suitable version "2.9", minimum required is "2.9")
-- THRIFT_HOME:
-- Thrift version: Thrift version 0.9.1
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
THRIFT_INCLUDE_DIR (ADVANCED)
   used as include directory in directory <FILE_LOCATION>
   used as include directory in directory <FILE_LOCATION>
   used as include directory in directory <FILE_LOCATION>
   used as include directory in directory <FILE_LOCATION>
   used as include directory in directory <FILE_LOCATION>

-- Configuring incomplete, errors occurred!
`

--- a/README.md
+++ b/README.md
@@ -18,9 +18,12 @@ make
 **Ubuntu 16.04**
 
 ```sh
-sudo apt-get install g++ cmake fuse libfuse-dev libthrift-dev python-thrift
+sudo apt-get install software-properties-common
+sudo add-apt-repository ppa:scylladb/ppa
+sudo apt-get update
+sudo apt-get install g++ cmake fuse libfuse-dev python-thrift thrift-compiler scylla-libthrift010-dev libboost-dev
 thrift -r --gen cpp --gen py server.thrift
-cmake CMakeLists.txt
+cmake -DCMAKE_PREFIX_PATH=/opt/scylladb CMakeLists.txt
 make
 ```
 

